README.md

```diff
@@ -100,64 +34,53 @@ jobs:
           level: warning
 ```
 
-## Advanced use cases
+## Inputs
 
-This action can be combined with [peter-evans/create-pull-request](https://github.com/peter-evans/create-pull-request) or [stefanzweifel/git-auto-commit-action](https://github.com/stefanzweifel/git-auto-commit-action) to also apply the annotated changes to the repository.
+### `github_token`
 
-### Commit changes
+**Required**. Must be in form of `github_token: ${{ secrets.github_token }}`'.
 
-```yaml
-name: reviewdog
-on: [pull_request]
-jobs:
-  name: runner / black
-  runs-on: ubuntu-latest
-  steps:
-    - uses: actions/checkout@v2
-      with:
-        ref: ${{ github.head_ref }}
-    - name: Check files using black formatter
-      uses: reviewdog/action-black@v1
-      with:
-        github_token: ${{ secrets.GITHUB_TOKEN }}
-        reporter: github-check
-        level: error
-        fail_on_error: true
-        format: true
-    - name: Commit black formatting results
-      if: failure()
-      uses: stefanzweifel/git-auto-commit-action@v4
-      with:
-        commit_message: ":art: Format Python code with psf/black push"
-```
+### `workdir`
 
-### Create pull request
+**Optional**. The directory to run remark-lint in. Defaults to `.`.
 
-```yaml
-name: reviewdog
-on: [pull_request]
-jobs:
-  name: runner / black
-  runs-on: ubuntu-latest
-  steps:
-    - uses: actions/checkout@v2
-    - name: Check files using black formatter
-      uses: reviewdog/action-black@v1
-      with:
-        github_token: ${{ secrets.GITHUB_TOKEN }}
-        reporter: github-check
-        level: error
-        fail_on_error: true
-        format: true
-    - name: Create Pull Request
-      if: failure()
-      uses: peter-evans/create-pull-request@v3
-      with:
-        token: ${{ secrets.GITHUB_TOKEN }}
-        title: "Format Python code with psf/black push"
-        commit-message: ":art: Format Python code with psf/black"
-        body: |
-          There appear to be some python formatting errors in ${{ github.sha }}. This pull request
-          uses the [psf/black](https://github.com/psf/black) formatter to fix these issues.
-        branch: actions/black
-```
+### `format`
```

Comment:
Remove?

Reply:
Good catch I totally overlooked those two arguments in the documentation.